### PR TITLE
updated README for several packages

### PR DESCRIPTION
**Pull request by dsawali**
Updated README for these following packages:
- taquito-beacon-wallet
- taquito-http-utils
- taquito-ledger-signer
- taquito-local-forging

Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: packages/taquito-ledger-signer/README.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+Install the package as follows
+```
+npm install @taquito/ledger-signer
+```
+
+## Usage
+### Prerequisites
+To use the Ledger Signer we must first import the desired transport from the [LedgerJs library](https://github.com/LedgerHQ/ledgerjs).
 
+The Ledger Signer has currently been tested with `@ledgerhq/hw-transport-node-hid` for Node-based applications and with `@ledgerhq/hw-transport-u2f` for web applications.
```
Comment (roxaneletourneau) [inline packages/taquito-ledger-signer/README.md:18]: `@ledgerhq/hw-transport-u2f` is deprecated and be replace by `@ledgerhq/hw-transport-webhi`

Comment (dsawali) [reply packages/taquito-ledger-signer/README.md:18]: thank you for the headsup, will resolve.

File: packages/taquito-http-utils/README.md [hunk 2]
```diff
@@ -1,21 +1,48 @@
...
\ No newline at end of file
+THIS SOFTWARE IS PROVIDED "AS IS" AND ANY EXPRESSED OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE REGENTS OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.# Taquito HTTP Utilities package
```
Comment (michaelkernaghan) [inline packages/taquito-http-utils/README.md:48]: # Taquito HTTP Utilities package  seems out of place here

File: packages/taquito-ledger-signer/README.md [hunk 3]
```diff
@@ -13,14 +26,13 @@ const transport = await TransportU2F.create();
...
 
-The constructor of the `LedgerSigner` class can take three other parameters. If none are specified, the default values are used.
+The constructor of the `LedgerSigner` class takes three other optional parameters. If none are specified, the following default values are used:
 
-- path: **default value is "44'/1729'/0'/0'"**  
-  You can use as a parameter the `HDPathTemplate` which refers to `44'/1729'/${account}'/0'`. You have to specify what is the index of the account you want to use. Or you can also use a complete path as a parameter.  
-  _More details about paths below_
-- prompt: **default is true**  
+- `path`: **default value is "44'/1729'/0'/0'"**  
+  You can use as a parameter the `HDPathTemplate` which refers to `44'/1729'/${account}'/0'`. You have to specify what is the index of the account you want to use. Or you can also use a complete path as a parameter. More details about paths [here](https://tezostaquito.io/docs/ledger_signer#derivation-paths-hd-wallet--bip-standards)
```
Comment (michaelkernaghan) [inline packages/taquito-ledger-signer/README.md:32]: specify what is the index ->  specify the index

**Review by roxaneletourneau — APPROVED** [review]

File: packages/taquito-http-utils/README.md (hunk 2 above)
Comment (dsawali) [reply packages/taquito-http-utils/README.md:48]: good catch, that was mistakenly copy-pasted

File: packages/taquito-ledger-signer/README.md (hunk 3 above)
Comment (dsawali) [reply packages/taquito-ledger-signer/README.md:32]: ah this was leftover from the previous doc, will amend

**Review by roxaneletourneau — APPROVED** [review]

**Pull request merged**